_posts/2017/05/2017-05-04-windows-installer.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Plans for Windows Installer"
+date: 2017-05-04
+time: "16:00:00"
+category: ["Community","Tools","Teaching"]
+---
+
+In March,
+we had two email threads at our [discuss mailing list](http://lists.software-carpentry.org/listinfo/discuss)
+related with nano on Windows machine.
+The [first thread](http://lists.software-carpentry.org/pipermail/discuss/2017-March/005085.html)
+was about "nano not found",
+a bug on our installer that we never manage to trace.
+The [second thread](http://lists.software-carpentry.org/pipermail/discuss/2017-March/005140.html)
+was about some "misbehaviour" of nano on Windows
+and the suggestion to use [Atom](https://atom.io/) as the default text editor.
+The suggestion to use Atom
+started a long discussion where instructors manifest their reasons to use nano
+and why Atom is "evil" which lead us to start investigating ways to install nano properly on Windows.
+On May 3rd,
+Kate Hertweck, Maneesha Sane, Naupaka Zimmerman, Rémi Emonet and I met online
+to draft a plan---taking in consideration all the feedback that was provided in the past month---to solve the problem with nano on Windows.
+
+## Plan
+
+We will work to use [Git for Windows SDK](https://github.com/git-for-windows/git/wiki/Technical-overview)
+to compile our own installer that will include
+bash, Git, nano, SQLite and make.
+In the future we can work to include man pages and Jekyll.
+
+## Timeline
+
+- YYYY-MM-DD: Publish Plan
+- YYYY-MM-DD: Beta version of the installer is release
+- YYYY-MM-DD: Instructors test the installer
+- YYYY-MM-DD: First version of the installer is release
+- YYYY-MM-DD: Workshop template is update to use the installer
+- YYYY-MM-DD: Report back from first 3 workshops using the new installer
```

Comment:
I think you mean add dates for these tasks? I can take a try, and if they aren't reasonable, feel free to change them.